ACE/ace/OS_NS_Thread.cpp

```diff
@@ -4806,6 +4807,16 @@ ACE_OS::unique_name (const void *object,
 }
 #endif
 
+pid_t
+ACE_OS::thr_gettid ()
+{
+#if defined (ACE_LINUX) && defined (ACE_HAS_GETTID)
```

Comment:
Is a check of just ACE_HAS_GETTID not enough?

Reply:
At the moment it would be. `syscall(SYS_gettid)` is basically the only way to use gettid, but is very specific to Linux. I was thinking of `ACE_HAS_GETTID` also being able to be defined if another platform had gettid that someone wanted to had.